### ISSUE TITLE
Cannot read stackoverflow data dump XML

### ISSUE THREAD

**Issue by ebuildy** [databricks/spark-xml#109]
I would like to parse XML from stackoverflow data dump, here an example of row:

``` xml
<row Id="7" PostTypeId="2" ParentId="4" CreationDate="2008-07-31T22:17:57.883" Score="287" Body="An explicit cast to double isn't necessary." OwnerUserId="9" LastEditorUserId="967315" LastEditDate="2012-10-14T11:50:16.703" LastActivityDate="2012-10-14T11:50:16.703" CommentCount="0" />
```

But loading this XML, nothing happen (no error, and schema is empty), is it compatible with this library?

Thanks you,

**Comment by HyukjinKwon** [comment]
To be clear, it is a duplicate of https://github.com/databricks/spark-xml/issues/92 but thanks for some cases I would definitely try.

**Comment by ebuildy** [comment]
D'oh sorry for the duplicate issue, didn't find it yesterday. Do you want to me to close this issue?

Thanks you,

**Comment by HyukjinKwon** [comment]
I think it is fine. I think it is a good reminder to test stackoverflow data dump.

**Comment by queirozfcom** [comment]
I still can't parse the dump. I think this is related to https://github.com/databricks/spark-xml/issues/190.

**Comment by BioQwer** [comment]
fix in #321
